plugins/poll/assets/javascripts/discourse/templates/modal/poll-ui-builder.hbs

```diff
@@ -17,6 +17,16 @@
                     valueAttribute="value"}}
       </div>
 
+      {{#unless isNumber}}
+        <div class="input-group poll-select">
+          <label class="input-group-label">{{i18n 'poll.ui_builder.poll_chart_type.label'}}</label>
+          {{combo-box content=pollChartTypes
+                      value=chartType
+                      allowInitialValueMutation=true
```

Comment:
this is going to be deprecated.

The preferred solution is to set `chartType` with an initial value

Reply:
@jjaffeux the initial value of `chartType` is set in the controller. What do you mean be "this" being deprecated?

Reply:
I mean I will remove "allowInitialValueMutation" option from select-kit, this is not a good practice.

So if `chartType` is set, why do you need this? `allowInitialValueMutation` is basically doing:

```
if (!this.chartType) {
  this.set("chartType", pollChartTypes.firstObject);
}
```

Reply:
Yeah, I had no clue what you were referring to.

All the other combo-boxes have that key value pair, and I copy pasted without thinking much about it. Happy to remove it

Reply:
Yes I know all of this is not documented ATM, and a for a reason, there were a lot of weird options to handle all the cases we had and all of this will be gone :)

farewell